add missing include to fix compiler errors like:

    error: variable std::ofstream summary_out has initializer but incomplete type

see also https://github.com/nanoporetech/dorado/pull/780

author: Kenneth Hoste (HPC-UGent)
--- dorado/dorado/cli/duplex.cpp.orig	2024-04-30 17:59:15.483935823 +0200
+++ dorado/dorado/cli/duplex.cpp	2024-04-30 17:59:34.658694274 +0200
@@ -39,6 +39,7 @@
 #include <cstdlib>
 #include <exception>
 #include <filesystem>
+#include <fstream>
 #include <memory>
 #include <thread>
 #include <unordered_set>
--- dorado/dorado/cli/demux.cpp.orig	2024-04-30 18:13:40.327122548 +0200
+++ dorado/dorado/cli/demux.cpp	2024-04-30 18:15:37.576760942 +0200
@@ -17,6 +17,7 @@
 #include <spdlog/spdlog.h>
 
 #include <chrono>
+#include <fstream>
 #include <memory>
 #include <optional>
 #include <string>
